### PR TITLE
decode-html-ack-policy-coverage

### PR DESCRIPTION
**Pull request by scottf**

### CODE REVIEW

File: src/examples/java/io/nats/examples/NatsJsPullFetch.java [hunk 1]
```diff
@@ -129,15 +130,15 @@ public static void main(String[] args) {
...
+            messages = sub.fetch(10); //, Duration.ofSeconds(3));
             report(messages);
             System.out.println("We should have received 10 message, we received: " + messages.size());
             sleep(3000); // longer than the ackWait
 
             // 9. Fetch messages,
             // -  get the 10 messages we didn't ack
             System.out.println("----------\n9. Fetch, get the messages we did not ack.");
-            messages = sub.fetch(10, Duration.ofSeconds(3));
+            messages = sub.fetch(10); //, Duration.ofSeconds(3));
```
Comment (ColinSullivan1) [inline src/examples/java/io/nats/examples/NatsJsPullFetch.java:141]: Did you mean to leave these comments in?

File: src/main/java/io/nats/client/impl/ConsumerInfo.java [hunk 2]
```diff
@@ -1,152 +1,153 @@
...
-                ", numAckPending=" + numAckPending +
-                ", numRedelivered=" + numRedelivered +
-                ", created=" + created +
-                ", Delivered" + delivered +
-                ", AckFloor" + ackFloor +
-                ", " + configuration +
-                '}';
-    }
-}
+// Copyright 2020 The NATS Authors
```
Comment (ColinSullivan1) [inline src/main/java/io/nats/client/impl/ConsumerInfo.java:1]: Why the move?   I'd think ConsumerInfo would be a package level class alongside the interfaces...

Comment (scottf) [reply src/main/java/io/nats/client/impl/ConsumerInfo.java:1]: I needed access to things in the impl package.

**Review by ColinSullivan1 — APPROVED** [review]
LGTM.

**Pull request merged**